sklearn/linear_model/tests/test_logistic.py

```diff
@@ -2076,8 +2076,7 @@ def test_liblinear_not_stuck():
         clf.fit(X_prep, y)
 
 
-@pytest.mark.usefixtures("enable_slep006")
-def test_lr_cv_scores_differ_when_sample_weight_is_requested():
+def test_lr_cv_scores_differ_when_sample_weight_is_requested(enable_slep006):
```

Comment:
the reason I'm using `usefixtures` is that it makes it more explicit and more readable. When `enable_slep006` is passed in the function signature, the reader of the test will see an argument which is not used in the test at all, and therefore could maybe be removed. For fixtures which are not used in the test, and rather have a side effect on how the test is run, I find the decorator syntax more readable and explicit.

Reply:
Okay, lets go with `usefixtures`.